tests/rptest/services/redpanda.py

```diff
@@ -913,6 +913,11 @@ def pids(self, node):
     def started_nodes(self):
         return self._started
 
+    def get_nodes(self) -> list[RedpandaNode]:
+        return [
+            RedpandaNode(n.account.hostname, self.idx(n)) for n in self.nodes
+        ]
```

Comment:
What purpose does `RedpandaNode` serve other than to save typing out `n.account.hostname`? I kind of think it leads to confusion because then we may have a reference to a node but without the normal interface. Perhaps use `RedpandaNode` internally to your tests or submit a patch to `redpanda-data/ducktape` to add a property accessor easier access to `hostname`?

Reply:
I was briefly confused above when I saw `node.ip` above and thought "huh, does ClusterNode have an ip method now?".  Let's avoid having a similar-but-different node class.

Reply:
ip isn't a problem but I wanted to have id property which is Redpanda specific so it's missing in ClusterNode. I'm going to move `get_nodes` to Admin